src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -158,7 +129,7 @@ object AttributeConversions {
     case AttributeBoolean(b) => b
     case AttributeNumber(n) => n.bigDecimal
     case AttributeString(s) => s
-    case AttributeValueList(l) => l.map(attributeToProperty(_)).asJava
```

Comment:
Lists are now handled separately in ```GraphDAO.serializeAttributeMap```. It's not needed here because ```AttributeValueList``` is no longer an ```AttributeValue```. 

Although I'm not sure why we moved the conversion code out of ```GraphDAO``` in the first place, since it just makes things confusing.